bluesky_queueserver/manager/comms.py

```diff
@@ -3,6 +3,8 @@
 import json
 import asyncio
 import uuid
+import zmq
+import zmq.asyncio
```

Comment:
This additional import should help with tests on Mac. Some relevant info: https://github.com/zeromq/pyzmq/issues/1314#issuecomment-512753984.